include/tables/scene_table.h

```diff
@@ -110,6 +110,7 @@
 /* 0x62 */ DEFINE_SCENE(spot18_scene, g_pn_41, SCENE_GORON_CITY, SDC_GORON_CITY, 0, 0)
 /* 0x63 */ DEFINE_SCENE(spot20_scene, g_pn_42, SCENE_LON_LON_RANCH, SDC_LON_LON_RANCH, 0, 0)
 /* 0x64 */ DEFINE_SCENE(ganon_tou_scene, g_pn_43, SCENE_OUTSIDE_GANONS_CASTLE, SDC_OUTSIDE_GANONS_CASTLE, 0, 0)
+#if OOT_DEBUG
 // Debug-only scenes
 /* 0x65 */ DEFINE_SCENE(test01_scene, none, SCENE_TEST01, SDC_CALM_WATER, 0, 0)
```

Comment:
guess you can remove the comment, and I would linebreak before the `#if` personally but just preference

Reply:
Removed the comment, but I think I prefer without the line break, it looks more like it's one table if OOT_DEBUG is true (similar to what we've been doing in functions.h)